src/screens/HomeScreen.js

```diff
@@ -24,6 +24,130 @@ import { ScreenName } from '../types';
 
 const { MATOMO_TRACKING, ROOT_ROUTE_NAMES } = consts;
 
+const renderItem = ({ item }) => {
+  const {
+    buttonTitle,
+    categoriesNews,
+    fetchPolicy,
+    showData,
+    navigate,
+    navigation,
+    query,
+    queryVariables,
+    showVolunteerEvents,
+    title
+  } = item;
+
+  const NAVIGATION = {
+    CATEGORIES_INDEX: {
+      name: ScreenName.Index,
+      params: {
+        title,
+        query: QUERY_TYPES.CATEGORIES,
+        queryVariables: {},
+        rootRouteName: ROOT_ROUTE_NAMES.POINTS_OF_INTEREST_AND_TOURS
+      }
+    },
+    EVENT_RECORDS_INDEX: {
+      name: ScreenName.Index,
+      params: {
+        title,
+        query: QUERY_TYPES.EVENT_RECORDS,
+        queryVariables: { limit: 15, order: 'listDate_ASC' },
+        rootRouteName: ROOT_ROUTE_NAMES.EVENT_RECORDS
+      }
+    },
+    NEWS_ITEMS_INDEX: ({
+      categoryId,
+      categoryTitle,
+      categoryTitleDetail,
+      indexCategoryIds,
+      rootRouteName
+    }) => {
+      const queryVariables = { limit: 15 };
+
+      if (indexCategoryIds?.length) {
+        queryVariables.categoryIds = indexCategoryIds;
+      } else {
+        queryVariables.categoryId = categoryId;
+      }
+
+      return {
+        name: ScreenName.Index,
+        params: {
+          title: categoryTitle,
+          titleDetail: categoryTitleDetail,
+          query: QUERY_TYPES.NEWS_ITEMS,
+          queryVariables,
+          rootRouteName: rootRouteName || ROOT_ROUTE_NAMES.NEWS_ITEMS
+        }
+      };
+    }
+  };
+
+  if (!showData) {
+    return null;
+  }
+
+  if (showData && !!categoriesNews?.length) {
```

Comment:
this `showData` is not needed anymore, as it is always `true` when the code reaches this line

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/81aa9283fae19a01bde21a5b5012d7b7bdab7531